src/ontology/mondo-edit.obo

```diff
@@ -233095,11 +233109,21 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/10
 
 [Term]
 id: MONDO:0011905
-name: obsolete familial hypercholanemia
-property_value: IAO:0000231 IAO:0000229
-property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/2940 xsd:string
-is_obsolete: true
-replaced_by: MONDO:0031446
+name: familial hypercholanemia
```

Comment:
This is odd - a previously obsolete class gets re-instated? Afaik this should not really be done, but if that is common practice...

Reply:
no, this should not be reinstated. That is the problem with this PR, it is reinstated for some reason and it shouldn't be.

Reply:
I re-obsoleted this (I looked at the file on master and copied and pasted the lines into this PR)